test/support/routers/original.ex

```diff
@@ -0,0 +1,27 @@
+defmodule ExampleWeb.Router do
```

Comment:
Why exactly is this being added? I didn't catch where it was referenced above...

Reply:
You have to add a `resources` line to your router after running `mix torch.gen.html`. For the integration test in the example app, I decided to just copy in a pre-modified router before running tests, and then replace it with the unmodified router.